_common

```diff
@@ -36,7 +37,7 @@ runcli() {
     rc=$?
     set -e
     [[ "$rc" == 0 ]] && echo "$output" && return
-    warn "openqa-cli ($(caller)): (${args[*]}) rc: $rc >>>$output<<<"
+    warn "openqa-cli ($(caller "$runcli_caller_level")): (${args[*]}) rc: $rc >>>$output<<<"
```

Comment:
I was hoping for something more generic in the `try` function, e.g. in perl you can do:
```
some test_helper_function {
    local $Test::Builder::Level = $Test::Builder::Level + 1;
    ...
}
```
The error message including the caller we wouldn't just need for the `runcli` function, but I would expect the right caller level in every function we test.
OTOH the level just wrong in the tests, so I'd prefer to remove this commit again.

Reply:
Maybe it could be done by adding a wrapper around the `caller` function that uses such a `level` variable, and `try` will increase it.

Reply:
That strategy worked in b4455c0.